go-controller/README.md

```diff
@@ -95,6 +95,11 @@ Values for some configuration options can be passed in environment variables.
      the absolute path to the Kubernetes API CA certificate overriden by --k8s-cacert
   -K8S_TOKEN
      the Kubernetes API authentication token overriden by --k8s-token
+  -OVN_EMPTY_LB_EVENTS
```

Comment:
@putnopvut so it's totally not clear, but the env vars here are those interpreted by the binary itself, not the `ovnkube.sh` script. So you can remove this change.